### PR TITLE
Added credits page

### PR DESCRIPTION
**Pull request by muuvmuuv**
On this page, we can give credit to authors. I added those two for the new logo.

### CODE REVIEW

**Comment by muuvmuuv** [comment]
@Glavin001 done. I‘m working on a new website design, should I open an issue for that or just a pr when I‘m done (or partly)?

**Comment by muuvmuuv** [comment]
Oops I accidentally closed it...

**Comment by Glavin001** [comment]
@muuvmuuv I'd prefer to hold out for a big change until your redesign is implemented:

image 

Then we can do everything together, such as I'll go update the logo in other appropriate places, too.

You can implement your redesign in this PR.

**Comment by Glavin001** [comment]
@muuvmuuv I created an issue to discuss the logo: https://github.com/Unibeautify/website/issues/120
And another for the website design: https://github.com/Unibeautify/website/issues/121

From here on, please use GitHub's smart comments in your commit messages: https://help.github.com/articles/closing-issues-using-keywords/
For example, `Close #120. Updated the logo` and `See #121. Work on website redesign` then `Close #121. Complete website redesign`, etc.

File: website/data/users.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 - caption: "Unibeautify"
-  image: "img/unibeautify.png"
+  image: "img/signet.svg"
```
Comment (Glavin001) [inline website/data/users.yml]: I'd prefer the name be `img/unibeautify.svg`.

**Comment by Glavin001** [comment]
Closing as a duplicate of https://github.com/Unibeautify/website/pull/122 . Will continue there.

**Comment by Glavin001** [comment]
@muuvmuuv Please merge this branch into your other PR, if you have not already. Thanks!

**Pull request closed without merging**